.github/workflows/release.yaml

```diff
@@ -52,3 +52,51 @@ jobs:
         with:
           credential: ${{ secrets.PUB_CREDENTIAL_JSON }}
           package_directory: ./packages/iconoir-flutter
+      - name: Build
+        run: ./node_modules/.bin/next build
+        working-directory: iconoir.com
+      - name: Export
+        run: ./node_modules/.bin/next export
+        working-directory: iconoir.com
+      - name: Setup Pages
+        uses: actions/configure-pages@v1
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v1
+        with:
+          path: './iconoir.com/out'
```

Comment:
Isn't this already done in the separate "website" job?

Reply:
That was a temporary job for the PR; had to merge with the main `release.yaml` in order to make sure the releases were complete before rebuilding the website (otherwise the version number won't update properly).